packages/node-core/src/indexer/worker/worker.builder.ts

```diff
@@ -122,7 +122,12 @@ abstract class WorkerIO {
           args,
         });
       } catch (e) {
-        this.logger.error(e, `Failed to post message, function="${fnName}", args="${JSON.stringify(args)}"`);
+        this.logger.error(
```

Comment:
I think maybe we're better off extending bigint so we can do it globally

`BigInt.prototype.toJSON = function() { return this.toString() + 'n' }`